src/backend/libs/efiling-account-client/src/main/java/ca/bc/gov/open/jag/efilingaccountclient/CsoAccountServiceImpl.java

```diff
@@ -28,12 +31,15 @@ public CsoAccountDetails getAccountDetails(String userGuid) {
 
         try {
 
-            AccountFacadeBean accountFacadeBeanPort = accountFacade.getAccountFacadeBeanPort();
-            List<ClientProfile> profiles = accountFacadeBeanPort.findProfiles(userGuid);
 
+
+            List<ClientProfile> profiles = accountFacadeBean.findProfiles(userGuid);
+            //An account must only one profile associated to proceed
             if (profiles.size() == 1) {
                 ClientProfile profile = profiles.get(0);
                 csoAccountDetails = new CsoAccountDetails(profile.getAccountId(), profile.getClientId());
+            } else if (profiles.size() > 1) {
```

Comment:
what if profiles == 0?

Reply:
This case is for multiple profiles only. Other return null